aptos-move/aptos-release-builder/data/release-v1.4.yaml

```diff
@@ -0,0 +1,23 @@
+---
+testnet: true
+remote_endpoint: ~
+is_multi_step: true
+update_sequence:
+  - Framework:
+      bytecode_version: 6
+      git_hash: ~
+  - FeatureFlag:
+      enabled:
+        - code_dependency_check
```

Comment:
Why do we list all of these features out? We only need

ed25519_pubkey_validate_return_false_wrong_length
struct_constructors

Reply:
did we change to create new yaml file for every releases? I thought we'd keep one release.yaml file and always use it as source of truth.